src/figuresClasses.ts

```diff
@@ -1,19 +1,86 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
```

Comment:
Create seperate type Color and use it everywhere ypu need. No need to use union in all places like `'red' | 'green' | 'blue'`